pkg/types/azure/validation/platform.go

```diff
@@ -20,6 +20,10 @@ func ValidatePlatform(p *azure.Platform, publish types.PublishingStrategy, fldPa
 	}
 	if p.DefaultMachinePlatform != nil {
 		allErrs = append(allErrs, ValidateMachinePool(p.DefaultMachinePlatform, fldPath.Child("defaultMachinePlatform"))...)
+		allErrs = append(allErrs, ValidateDefaultDiskType(p.DefaultMachinePlatform, fldPath.Child("defaultMachinePlatform"))...)
+		// if p.DefaultMachinePlatform.OSDisk.DiskType == "UltraSSD_LRS" {
```

Comment:
commented code related to ultra ssds should be moved to a separate PR, which could be opened immediately.

Reply:
I keep searching for commented code and this keeps eluding me. Thanks for pointing it out.